lumigator/python/mzai/backend/backend/main.py

```diff
@@ -51,13 +52,9 @@ def create_app() -> FastAPI:
     app = FastAPI(**LUMIGATOR_APP_TAGS)
 
     # Adding CORS middleware
-    origins = [
-        "http://localhost",
-        "http://localhost:3000",
-    ]
     app.add_middleware(
         CORSMiddleware,
-        allow_origins=origins,  # Adjust this list as needed for security (e.g., ["http://localhost:3000"])
+        allow_origins=["*"],
```

Comment:
Can this be done as an environment variable that by default is a `*`?  so we can allow users to customize it.

Reply:
Yep, good idea.